README.md

```diff
@@ -48,6 +48,28 @@ shut down immediately. They leave no traces (outside of `/nix/store`) on the hos
   NixOps can be used to deploy to various other backends like cloud providers.\
   Requires: [Nix](https://nixos.org/nix/), [VirtualBox](https://www.virtualbox.org)
 
+#### Tests
+The internal test suite is also useful for exploring features.  
```

Comment:
nit: trailing whitespace

Reply:
The trailing space is markdown syntax to force a line-break.

Reply:
TIL